packages/editor-ui/src/plugins/i18n/locales/en.json

```diff
@@ -828,6 +828,21 @@
 		},
 		"notBeingUsed": "Not being used"
 	},
+	"template": {
+		"buttons": {
+			"goBackButton": "Go back",
+			"useThisWorkflowButton": "Use this workflow"
+		},
+		"details": {
```

Comment:
not sure if we need this nesting here. @ivov can you give feedback here? this is the expected view
<img width="321" alt="Screenshot 2021-12-24 at 13 23 43" src="https://user-images.githubusercontent.com/4711238/147352358-f14cb307-cb69-421f-af9a-4a8f4bdb6314.png">